docs/tutorials/working_with_large_catalogs.ipynb

```diff
@@ -2,22 +2,395 @@
  "cells": [
```

Comment:
<p>Maybe specify that this will be the points that match both filters, instead of either filter?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/236/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>